react-ui/src/utils/constants.js

```diff
@@ -35,6 +35,22 @@ export const RULE_DIFF_PROPERTIES = [
   'update_type',
   'version',
 ];
+// Products that use only a subset of the available Rule fields
+// should define them here.
+export const RULE_PRODUCT_UNSUPPORTED_PROPERTIES = {
+  Guardian: [
```

Comment:
👍

Reply:
Oh sorry, I should've mentioned that code has all been reviewed already over in the `balrog-ui` repo...